apps/video/js/video.js

```diff
@@ -1313,10 +1310,12 @@ function handleSliderTouchEnd(event) {
 
   dom.playHead.classList.remove('active');
 
+  // We didn't move the play head while the user's finger was on it
+  // so now it may be in the wrong place.
+  updateVideoControlSlider();
```

Comment:
Is this change is left over from when the patch was not using the 'seeker'? Now that you are seeking while the slider is being dragged it seems this code is not needed.